### PR TITLE
Related #4927: Add ss58 address format network identifier prefix for DataHighway parachain

### PR DESCRIPTION
**Pull request by ltfschoen**
**UPDATE: 26 May 2020** Changed to request ss58 prefixes of 49, which has not already been reserved

- What does it do?
    * We would like to claim these ss58 prefixes:
      * 49: DataHighway Mainnet
    * We will be using Substrate's ss58 prefix of 42 for DataHighway Testnet
    * [DataHighway](https://github.com/DataHighway-DHX/node) will be a parachain supported by the Polkadot network. The chain's repo requires an ss58 address format network identifier prefix, see associated [issue](https://github.com/DataHighway-DHX/node/issues/87).

- What important points reviewers should know?
    * If this PR is approved, the ["External Address Format (SS58)" section of the Substrate wiki](https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)#address-type) should be updated to reflect the changes in this PR (i.e. add the DataHighway's ss58 address format network identifier prefix)

- My notes:
  - If this PR is approved, **I will need to update**
      * Polkadot.js util-crypto's ss58: https://github.com/polkadot-js/common/blob/master/packages/util-crypto/src/address/ss58.ts
      * Polkadot.js's app-config sections: https://github.com/polkadot-js/apps/blob/master/packages/apps-config/README.md

### CODE REVIEW

**Comment by jacogr** [comment]
Thanks for the heads up, but don't manage specific user-chains myself, it is up to the chain devs. f they want support, see https://github.com/polkadot-js/apps/tree/master/packages/apps-config

**Comment by gavofyork** [comment]
The "Reserved" of `Reserved46` pertains to the fact that Parity is reserving it for future use. You might want to try claiming one of the indices which are not reserved.

**Comment by ltfschoen** [comment]
Thanks for the feedback, I've pushed a commit to request indice 49, which has not been reserved

**Comment by bkchr** [comment]
> Thanks for the feedback, I've pushed a commit to request indice 49, which has not been reserved

Please take a closer look: https://github.com/paritytech/substrate/blob/a4ba9bec6c8d58365de4d95bfdce1c51c007c316/primitives/core/src/crypto.rs#L479

**Comment by ltfschoen** [comment]
> substrate/primitives/core/src/crypto.rs

Ok, I've had a closer look, and I don't mean to overcomplicate things, but do you mean just to take a closer look at that comment?

So currently we're trying to ask for indice `49` since it doesn't appear to be reserved based on the following observations:
1. There isn't a `Reserved49` already in that file
2 I've checked that there aren't any PRs already asking for `49`
3 It's not one of the reserved indices (i.e. `1`, `3`, `43`, `46`, `47`, which have the word 'reserved' in their number and name)
4. I've checked that I've used a unique identifier, number, name, and description.
5. I've avoided choosing `48` since this comment `// Note: 48 and above are reserved.` says that `48` is a reserved indice, but I'm not sure why there isn't a line `Reserved48 => (48, "reserved48", "Reserved for future use (48).")`
6. It doesn't mention anywhere that we can't try to claim one of the indices that are greater that `48`
7. The comment `// Note: 48 and above are reserved.` mentions that `48` is reserved, even though there isn't a line `Reserved48 => (48, "reserved48", "Reserved for future use (48).")`, so I've assumed that since it says `... and above are reserved` that all other indices that are less than the number `48` are reserved, even if they too aren't shown as being reserved (i.e. there isn't a 
`Reserved13 => (13, "reserved13", "Reserved for future use (13).")` but I assume it's reserved as it satisfies the comment), so even though these indices don't appear to have been taken (i.e. `4`, `13` to `15`, `17`, `19` to `31`, `33` to `35`, `37` to `41`, `45`, and `48`), I've assumed that they are reserved and haven't chosen any of them.
8. I've reviewed other PRs that have requested an available indice successfully (i.e. https://github.com/paritytech/substrate/pull/5155/files, https://github.com/paritytech/substrate/pull/5061/files, and https://github.com/paritytech/substrate/pull/5506/files), and I've noticed that whilst they chose an indice that was less that or equal to `48` and didn't choose to replace one of the visibly reserved indices (i.e. `Reserved47`), I don't know if they claimed indices that they already knew were reserved for them, and which were one of the reserved indices mentioned in the comment  `// Note: 48 and above are reserved.`.

So despite the above assumptions, are my assumptions in points 6. and 8. incorrect, in which case I could modify my PR to be the following, where `<INDICE>` is either `4`, `13` to `15`, `17`, `19` to `31`, `33` to `35`, `37` to `41`, or `45`?
```
	DataHighwayAccount =>
		(<INDICE>, "datahighway", "DataHighway mainnet, standard account (*25519).")
```

**Comment by jacogr** [comment]
It is clear without having explicit entries for each of the reserved fields. `Note: 48 and above are reserved.` Unpacking -

- `48` is unavailable (it appears in the comment explicitly)
- `48 + n` is unavailable (it appears in the comment as "and above")

TL;DR Pick any unassigned number < 48 and push the change.

**Comment by ltfschoen** [comment]
Cheers, sorry I thought `and above` was referring to the lines of code above the comment, rather than indices above 48.

**Comment by kianenigma** [comment]
> TL;DR Pick any unassigned number < 48 and push the change.

updates @ltfschoen ?
